app/controllers/api/v1/profiles_controller.rb

```diff
@@ -68,6 +87,10 @@ def profile_params
         )
       end
 
+      def tag_params
+        params.permit(:name)
```

Comment:
It's not clear which name is that.
We need to explicitly say this attr is for tags.
so it should be `params.require(:tag).perfmit(:name)`

Reply:
```
params = {
  "tag": { "name": ... },
  "profile": { "name": ... }
}
```